### PR TITLE
CryPowder:: Enables S-Empty option and Polaris system test updated

### LINKED ISSUE #16772: CRY diffraction scripts: fix/enable "Sempty" option and add system test for POLARIS that uses that option
After some trial and error with Helen we found that the Mantid 3.6+ version of the Aziz scripts works well for current POLARIS data and reduction practices (some question marks still being investigated). One of the issues that we had to fix is that one line in the code that processes the option "Sempty" is commented out: https://github.com/mantidproject/mantid/blob/master/scripts/CryPowderISIS/cry_focus.py#L120
That option is always used for POLARIS, so it needs fixing.
That line should be uncommented. That is enough to get the option working properly.
We should add a system test that uses it. In our current system tests this option is always set to 0 - not used.

Check with me for the required files (runs, VanaPeaks, pref file, etc.).

### PR DESCRIPTION
**Pull request by shahrozahmed1**
Description of work.

The S-Empty option has been enabled within the the CryPowderISIS script, this option will be triggered when a S-Empty run number is provided within the `.pref` file. S-Empty option is not being used by any of the existing system tests which is running the CryPowderISIS script, however I have added additional system test using the files provided by Helen for the Polaris instrument. 
_NOTE: The `.pref` file provided by Helen seems to be using same run number for S-Empty and V-Empty._ 

**To test:**
Code Check
If possible run `ISISPowderDiffractionXXXXTest` locally
Check that system test build pass
No further pylint violation generated

Fixes #16772 
[Release Notes](https://github.com/mantidproject/mantid/commit/5df5d12a141d4c81c77c8b6b77a062754526ab6c)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [x] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [x] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [x] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/tests/analysis/ISISPowderDiffractionPolarisTest.py [hunk 1]
```diff
@@ -424,3 +424,185 @@ def test_datfile_with_workspace(self):
...
         self.assertAlmostEqual(dat_data[8].readE(0)[3369], dat_data[9].readE(0)[3369], places=DIFF_PLACES)
         self.assertAlmostEqual(300.4597, dat_data[9].readX(0)[10], places=DIFF_PLACES)
+
+
+# ================ Below test cases uses cycle 16_1 pref file ==================
+# =================== where 'ExistingV' = 'no' in pref file ===================
+
+class ISISPowderDiffractionPol3(stresstesting.MantidStressTest):
+    def requiredFiles(self):
+        filenames = []
```
Comment (DavidFair) [inline Testing/SystemTests/tests/analysis/ISISPowderDiffractionPolarisTest.py]: Could this be initialized on deceleration?

**Comment by DavidFair** [comment]
@FedeMPouzols I've checked this over and the system tests pass on my end too. I'm happy to ship if you could check it over and make sure there isn't anything I have missed?

**Comment by FedeMPouzols** [comment]
@DavidFair: yes this looks good for me. The release note looks good as well. Put the squirrel when you're happy with it.

**Comment by DavidFair** [comment]
Ok I'm happy to :shipit:

**Pull request merged**